test/functional/feature_block.py

```diff
@@ -116,7 +122,38 @@ def run_test(self):
         b2 = self.next_block(2, spend=out[1])
         self.save_spendable_output()
 
-        self.sync_blocks([b1, b2])
+        self.sync_blocks([b1, b2], timeout=4)
+
+        # Pop the first unspent output in `out`.
+        spend_tx = out.pop(2)
+
+        # Submit blocks for rejection, each of which contains a single transaction
+        # (aside from coinbase) which should be considered invalid.
+        for TxTemplate in invalid_txs.iter_all_templates():
+            template = TxTemplate(spend_tx=spend_tx)
+
+            # Something about the serialization code for missing inputs creates
```

Comment:
Maybe put the test names here so someone can look for it in the future?

Reply:
Not sure what you mean by this - which test names?